libr/include/r_core.h

```diff
@@ -45,6 +45,13 @@ R_LIB_VERSION_HEADER(r_core);
 #define R_CORE_ANAL_KEYVALUE            16
 #define R_CORE_ANAL_JSON_FORMAT_DISASM  32
 
+#define R_CALLGRAPH_FORMAT_NO           0
```

Comment:
what about R_GRAPH_FORMAT insteado? RCallGraph is not a thing

Reply:
ok, fixed it